src/screens/ManageProfile.tsx

```diff
@@ -1,51 +1,129 @@
 import React from 'react'
 import { Button, Form } from 'react-bootstrap'
-import photo from '../assets/images/react.svg'
-// page for update user info firstname, lastname, sex, birthday, email and profile picture
+import { useState } from 'react'
+const url = import.meta.env.VITE_API_URL
+
+// page for update user info firstname, lastname, sex, birthdate, email
 export default function ManageProfile() {
+  interface User {
+    firstname: string
+    lastname: string
+    sex: string
+    birthdate: string
+    email: string
+  }
+  const [error, setError] = useState<string>('')
```

Comment:
error has never been used you can see the template for using error to create new HTML element both in LoginRegister.tsx
![image](https://user-images.githubusercontent.com/88868804/218989199-6b067322-1336-48cf-b4e7-a6bdceeaa607.png)